src/VerificationCodeServiceProvider.php

```diff
@@ -19,6 +22,9 @@ public function boot()
         }
```

Comment:
Should we set a date for the migration instead of the current date like we do in 'laravel-mayday-monitor'?
To prevent the migration being added multiple times with different timestamps?

Reply:
the issue we had in 'laravel-mayday-monitor', was that not related to having multiple migrations that had to be published in a specific order? this is not the case here